packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec

```diff
@@ -0,0 +1,24 @@
+Name: foreman-obsolete-packages
+Version: 3.6
+Release: 1%{?foremandist}%{?dist}
```

Comment:
This results in `3.6-1.fm3_6.el8`. Is `foremandist` needed? Or should we simplify version it 1.0 and rely on foremandist?

Reply:
I guess I'd do the later. There is not much in here that would warrant versionlocking with the Foreman version.

Otherwise you gotta add that to the post-branching-bump and get rather needless bumps as the source never changes.

Reply:
Or would we even go to just `1.0-1` without any Foreman version? That's probably fine since it's unlikely we'll obsolete things in stable branches.

Reply:
Also true.